src/main/java/cn/nukkit/plugin/PluginManager.java

```diff
@@ -31,8 +30,10 @@ public class PluginManager {
 
     private final SimpleCommandMap commandMap;
 
+    @Getter
```

Comment:
Reduce the usage of lombok, this refactoring is not necessary, we only use lombok for necessary pure data class(POJO)